src/benchmark.cc

```diff
@@ -386,29 +407,56 @@ void State::FinishKeepRunning() {
 namespace internal {
 namespace {
 
+int base10_digit_count(int64_t n) {
```

Comment:
watch the naming (`Base10DigitCount`). Also, given this isn't on the critical path, just using `std::log10(n) + 1` should be good for all cases instead of the unrolled version.